middleware/middleware.js

```diff
@@ -0,0 +1,18 @@
+const jwt = require('jsonwebtoken');
+
+module.exports = function(req, res, next) {
+ const token = req.header('x-auth-token');
+ if (!token) {
+    console.log('here');
```

Comment:
usually you will want to remove logs like this before merging into master. Or, if you want to leave it, you could add some more detail, like: no auth token provided